lib/glueby/contract/rpc.rb

```diff
@@ -10,6 +10,14 @@ def client
       def configure(config)
         @rpc = Tapyrus::RPC::TapyrusCoreClient.new(config)
       end
+
+      def switch_wallet(wallet)
+        before = client.config[:wallet]
+        client.config[:wallet] = wallet
+        result = yield(client)
+        client.config[:wallet] = before
+        result
+      end
```

Comment:
`RPC` class also better to move `Internal` package.

Reply:
Yes, I will do that.

Reply:
Done at d8abf349cd298ee13232408698ea11e46ba63138

Reply:
Done at e711e67946f0b44beb6d62de4ed90d6249dd29d1